charts/testkube-api/executors.json

```diff
@@ -138,7 +138,10 @@
         "name": "jmeterd-executor",
         "executor": {
             "executorType": "job",
-            "image": "kubeshop/testkube-jmeterd-executor:820cd99",
+            "image": "kubeshop/testkube-jmeterd-executor:69fe019",
```

Comment:
shouldn't the tags be the same?

Reply:
No, I don't think so, Since we are using different images for master(Job) and slave pods.
The task for master job is to spawn the pods and run the `jmeter test command` as per the test while the image of slave pods is running the `jmeter-server` for each pod and waiting for the above jmeter test command to be run.